### ISSUE TITLE
Declarative event mapping for signals does not add a dash between signal type and name

### ISSUE THREAD

**Issue by MikeMitterer** [bwu-dart-graveyard/polymer_elements#142]
By reading the documentation here:
http://www.polymer-project.org/articles/communication.html#using-ltpolymer-signalsgt

I would expect that, if I send: 

``` dart
asyncFire('polymer-signal', detail: {'name': "foo", 'data': "Foo!"});
```

(Your sample (sender): https://github.com/ErikGrimes/polymer_elements/blob/master/example/src/polymer_signals/my_app.html)

the signal name is on-polymer-signal-foo an not on-polymer-signalfoo (dash is missing here)

``` html
<!-- ??? maybe wrong -->
<polymer-signals on-polymer-signalfoo="{{fooSignal}}"></polymer-signals>
```

``` html
<!-- What I expect + much nicer to read! -->
<polymer-signals on-polymer-signal-foo="{{fooSignal}}"></polymer-signals>
```

Your sample:
https://github.com/ErikGrimes/polymer_elements/blob/master/example/src/polymer_signals/my_app.html

**Comment by zoechi** [comment]
Thanks for reporting. This seems to be a bug. https://github.com/PolymerLabs/polymer-signals/blob/master/demo.html looks like your preferred example.
I'll take a look.

**Comment by zoechi** [comment]
I published 0.2.0-pre.5 that contains this fix.
